packages/ui/src/organisms/ProductCard/stories/ProductCard.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import type { Story, Meta } from '@storybook/react'
+import React from 'react'
+
+// Atoms
+import Price from '../../../atoms/Price'
+import Badge from '../../../atoms/Badge'
+// Product Card components
+import ProductCardComponent from '../ProductCard'
+import ProductCardImage from '../ProductCardImage'
+import ProductCardInfo from '../ProductCardInfo'
+import ProductCardHeader from '../ProductCardHeader'
+import ProductCardPrice from '../ProductCardPrice'
+import ProductCardTags from '../ProductCardTags'
+import ProductCardLink from '../ProductCardLink'
+import type { ProductCardProps } from '../ProductCard'
+import mdx from './ProductCard.mdx'
+
+const ProductCardTemplate: Story<ProductCardProps> = ({ testId }) => (
+  <ProductCardComponent testId={testId}>
+    <ProductCardImage>
+      <img
+        alt="A vintage camera"
+        src="https://storecomponents.vtex.app/assets/fit-in/480x480/center/middle/https%3A%2F%2Fstorecomponents.vtexassets.com%2Farquivos%2Fids%2F155481%2FFrame-3.jpg%3Fv%3D636793814536230000"
+      />
+    </ProductCardImage>
+    <ProductCardInfo>
+      <ProductCardHeader>
+        <h3>Vintage Top Camera</h3>
+      </ProductCardHeader>
+      <ProductCardPrice>
+        <Price
+          value={89.9}
+          variant="selling"
+          formatter={formatter}
+          style={{ textDecoration: 'line-through' }}
+        />
+        <Price value={68.9} variant="selling" formatter={formatter} />
+      </ProductCardPrice>
+      <ProductCardTags>
+        <Badge>15% OFF</Badge>
+      </ProductCardTags>
+    </ProductCardInfo>
+    <ProductCardLink>
```

Comment:
It's a little bit confusing this one. A card should be wrapped into a link and the `add to cart` button should be a button, right?

Reply:
@tlgimenes hmm, good point. Related to the card being wrapped into a link I think it makes sense and perhaps I should add this, but regard the `Add to Card` being a button I'm not sure because the agencies may want this button to be a redirect to another page, can't they? That way the button would act as a link...

Reply:
Not really. Usually, when the user clicks on the card the user is redirected to the PDP. When the user clicks on `add to cart` they want the product to be added to the cart, not navigate. You can check this behavior at `www.marinbrasil.com.br` or at `mercado.carrefour.com.br`

Reply:
@tlgimenes after checking on [Marin Brasil](https://www.marinbrasil.com.br/) site I found that somehow a `button` is being used inside an `a` tag (which is discouraged by HTML5 validations). In 0551a30 I implemented something close to that, but I'm not sure if it would be the best approach for our case. What do you think?

Reply:
Nice. I just didn't uderstand why adding that inline `style` option, but for me it's ok

Reply:
That's the approach to have the `a` tag wrapping the entire component. As _relative_ the component doesn't wrap very well, for example, the `button` tag is kept outside the link

Reply:
Nice. Just one more thing. Can you add the link to base.store using this component? I'm not sure how we can use this link with gatsby's link withtout ending up with two `<a>` tags on the final html

Reply:
@tlgimenes looking at the [Carrefour](https://mercado.carrefour.com.br/) site, noticed that the `button` is outside the `a` tag. I think it is the best we can do to get rid of the ProductCard's inline style and the other problems we can have regard the `absolute` positioning.